### ISSUE TITLE
Training 2-3 models, suggestions?

### ISSUE THREAD

**Issue by prakharpbuf** [CorentinJ/Real-Time-Voice-Cloning#1157]
Hi,
Great work with Real Time Voice Cloning! 
I'm going to train two or three models:
1. Everything from scratch using LibriTTS dataset.
2. Fine tune the pre trained model on 1 hour of my own voice.
3. Maybe also a model using the pretrained model and train it for a few additional 100-200k steps using more data from Mozilla Common Voice or something else.
I don't have a very fast memory (I'll be using external hard drives)
I have NVIDIA Quadro P2000 GPUs
But, to train each model, I'll use separate PCs with same specs so they all train in parallel.

Any suggestions on playing around with hparams, different ideas for training, or anything else?
All suggestions are welcomed and appreciated. 

Thanks!

**Comment by oops408** [comment]
try using model architecture (eg. location vs. contact-based) and loss functions as hparams and see if those help fine tune it. i'm trying out SGD optimization to see if that would improve the results. oh yeah, maybe pitch shifting would be interesting as well...
